exercises/scale-generator/canonical-data.json

```diff
@@ -15,7 +15,20 @@
           "input": {
             "tonic": "C"
           },
-          "expected": [ "C", "C#", "D", "D#", "E", "F", "F#", "G", "G#", "A", "A#", "B" ]
+          "expected": [
```

Comment:
I can't come up with a rule for why the chromatic scale starting at C should be on separate lines but F should be on one line. They have the same number of notes and the same sum of string lengths. What is the rule?

Reply:
Fixed